### ISSUE TITLE
[enhancement] Header for direct access to PoE 48V supply?

### ISSUE THREAD

**Issue by zeroping** [Connaxio/espoir-devboard#1]
I'm interested in using an Espoir for power and control of LED lighting, and the one thing I'd be missing would be some way to directly access 48VDC. The Espoir is especially useful, since it's already designed for the full 25W from 802.3at.

There are some commercially-available LED driver modules that take in 48VDC for driving constant-current 12V and 24V LED strips, for example, the Meanwell NLDD-H. A module like that could be placed on an add-on board with access to VPOS and VRTN, with (optioisolated?) control PWM control from the MikroBUS connector.

If this is too much scope creep, just say so, but if not, I might look to see if there's a good way to add the appreciate headers for this. I know the board routing is fairly tight already, but it should only be two or three pins. (PGOOD would be handy too.)

Are there any electrical safety concerns with doing this? I know using a non-isolated LED driver will result in a non-isolated LED strip, but would there be issues created for all Espoir boards? Isolation distances that would need to be adhered to?

**Comment by ma-lalonde** [comment]
> Are there any electrical safety concerns with doing this? I know using a non-isolated LED driver will result in a non-isolated LED strip,

This right here. I had debated whether I wanted to make the PoE voltage available, but decided not to, as the point of using standardized PoE is partly to have the isolation for safety concerns, especially for long cables, outdoors installations. I did not want to make it easy for anyone to accidentally bypass the isolation. Also, linking the logic side of the board to the non-isolated side while also maintaining isolation requires so many parts it is basically equivalent to half the board.

However, if you want to access the PoE voltage anyway, you can solder directly to the pins of C18, under the board. This gives you the cleanest and safest access to the PoE voltage.

**Comment by zeroping** [comment]
I was hoping for a nice clean pin header, but I understand not wanting to encourage people to use the unisolated supply. Olimex does this with the non-isolated version of their board, but big warning labels are definitely warranted. 

I don't think the isolation concerns are an issue in my use case, as I would be using an isolated LED driver module, and something as simple as an optoisolator to connect the logic side to that driver's control pins. My original idea was to try to make a stacking board with an LED driver, but maybe I need to just spin my own variant of the design with it all on one PCB.  And that's possible, since this is an OSHW project!

In short: it's your call, and just wanted to make you aware of use cases like mine. Thanks for the good answer.

Even without direct access to 48V, this project is great, and I'm still interested to get a couple when you launch the CrowdSupply run.
